cmd/create.go

```diff
@@ -200,49 +256,69 @@ func runCreate(cmd *cobra.Command, _ []string) error {
 			return err
 		}
 
-		fmt.Printf("[%d/%d] Host: %s, status: %s\n",
-			i+1, max, hostStatus.ID, hostStatus.Status)
-
 		if hostStatus.Status == "active" {
-			if !pro {
-				fmt.Printf(`Inlets OSS exit-node summary:
-  IP: %s
-  Auth-token: %s
-
-Command:
-  export UPSTREAM=http://127.0.0.1:8000
-  inlets client --remote "ws://%s:%d" \
-	--token "%s" \
-	--upstream $UPSTREAM
-
-To Delete:
-	inletsctl delete --provider %s --id "%s"
-`,
-					hostStatus.IP, inletsToken, hostStatus.IP, inletsControlPort, inletsToken, provider, hostStatus.ID)
+			if delTunnel == true {
+				sig := make(chan os.Signal)
+				done := make(chan bool)
+
+				signal.Notify(sig, syscall.SIGINT, syscall.SIGTERM)
+
+				go func() {
+					sigval, ok := <-sig
+					fmt.Printf("\n%v\n", sigval)
+					done <- true
+					if ok {
+						close(sig)
+					}
+				}()
+
+				fmt.Printf("Your IP is: %s\n", hostStatus.IP)
+
+				var err error = nil
+				ctx, cancelFunction := context.WithCancel(context.Background())
```

Comment:
@viveksyngh I would love to have your feedback on this. Even after passing a req context to checkServiceUp() we're not able to cancel our request and exit out.